Composer/packages/client/src/pages/design/styles.ts

```diff
@@ -103,9 +103,23 @@ export const formEditor = css`
   min-width: 300px;
 `;
 
+export const visualPanelHeaderContainer = css`
+  height: 65px;
+`;
+
+export const visualPanelHeaderLeft = css`
+  overflow: hidden;
+  float: left;
+  width: calc(100% - 100px);
```

Comment:
why dont you use ```flex``` so the browser itself decides how much width give to each side. With this width and float, we will still have the issue if ```show code``` needs more than 100px to render in a certain locale.

Reply:
@hatpick Make sense, updated now.